spec/type_aliases/sortlist_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
+describe 'Resolv::Sortlist' do
+  context 'with valid sortlist elements' do
+    it { is_expected.to allow_value([]) }
+    it { is_expected.to allow_value([
+      '111.111.111.111',
+      '2001:0db8:85a3:0000:0000:8a2e:0370:7334',
```

Comment:
Would like to see a test for an IPV6 CIDR address

Reply:
According to the [man page](https://linux.die.net/man/5/resolv.conf), `sortlist` doesn't accept CIDR―only "IP-address-netmask pairs."

> A sortlist is specified by IP-address-netmask pairs.  The netmask is optional and defaults to the natural netmask  of  the  net.   The IP address and optional network pairs are separated by slashes.

Reply:
My mistake using wrong terminology.  IPV6:IPV6 netmask

Reply:
[RFC 4921](https://tools.ietf.org/html/rfc4291#section-2.3) indicates that IPV6:IPV6 netmasks aren't a thing―IPv6 _only_ uses CIDR.

This made me curious if it's possible to define preferential IPv6 ranges using `sortlist`.  [Cursory Google results](https://unix.stackexchange.com/a/332595) suggest that `sortlist` doesn't support IPv6 unlike [RFC 3484](https://tools.ietf.org/html/rfc3484#section-6), which should be preferred.  I haven't found a more authoritative reference, except for [this documentation](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.hale001/ipv6d0061001762.htm) from IBM which explicitly states that―at least for the zOS resolver―`sortlist` is IPv4-only.